swarm/storage/mru/resource.go

```diff
@@ -889,64 +827,58 @@ func (self *Handler) update(ctx context.Context, name string, data []byte, multi
 	return key, nil
 }
 
-// Closes the datastore.
+// Close closes the datastore.
 // Always call this at shutdown to avoid data corruption.
-func (self *Handler) Close() {
-	self.chunkStore.Close()
+func (h *Handler) Close() {
+	h.chunkStore.Close()
```

Comment:
how was this not a problem before? 
If the chunkstore is shared, it is closed in swarm.Stop too no?

Reply:
Maybe a remnant from when MRU had its own store.